ai-service/db.py

```
@@ -0,0 +1,36 @@
+        metadata: Optional info like filename, function name.
+
+    Raises:
+        Exception: If database operation fails.
+    """
+
+    try:
+        ids = [f"chunk-{uuid.uuid4()}" for _ in range(len(chunks))]
+        collection.add(
+            documents=chunks,
+            embeddings=embeddings,
+            ids=ids
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)